### DISCUSSION TITLE
Gracenote Update V17 to V21/V22

### DISCUSSION THREAD

**Discussion by cirrussr20** [Mr-MIBonk/M.I.B._More-Incredible-Bash#211] [General]
I am trying to update from Gracenote V17 which was installed in the unit from factory. I can rerun the V17 upload using the downloads from mibsolution one but V21 or V22 fail at 35% with the message update interrupted. I have tried numerous SD cards but they all behave exactly the same. I have used the tool to make the FEC changes and have uploaded maps without a problem. Can anyone provide any ideas on how to fix. Pics attached
![2023-05-15 13 46 07](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/69537588/9fca050d-5775-46d6-8b0d-2c67133dd20f)
![2023-05-15 13 45 09](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/69537588/2627818c-3442-42e6-8efe-72658d78a788)
![2023-05-15 13 45 02](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/69537588/4210ab3d-d3e6-4db8-b463-ad1652645a7e)
![2023-05-15 13 44 55](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/69537588/284c2b6a-7c4f-423a-b306-01622c7f2d93)
![2023-05-15 12 40 30](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/69537588/13dd370b-661e-43a7-959e-fe9f9ae2d324)

**Comment by harman-f** [comment]
Strange one. The updates work, never have heard about issues so far. 
there is no error message at all?

**Reply by djskual** [reply]
> Strange one. The updates work, never have heard about issues so far. there is no error message at all?

same problem here: #71

**Comment by cirrussr20** [comment]
No error message other than the message about update being interrupted. Would I be able to try using AIO approach but I do not have the skill to modify your program? Is there any way to delete the existing Gracenote database before running the update? It's odd the V17 works fine from the same source.

**Comment by cirrussr20** [comment]
The #71 contains my post. Is there any other way you can think of to install the update?

I wondered if the file sizes were incorrect, and it was expecting a bigger file but they are all correct?

The maps updates are more important and work fine, I have seen some comments that V17 is different in some way to V21/V22?

I am happy to follow any ideas you may have.

Mike

**Comment by cirrussr20** [comment]
Here is a link to a video of what happens

https://www.dropbox.com/s/btqui55t0zo0eqq/2023-05-16%2016.43.50.mp4?dl=0

**Comment by harman-f** [comment]
Try this metainfo2.txt with V22, just replace it with the one from the download.

[metainfo2.txt](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/files/11490939/metainfo2.txt)

**Comment by cirrussr20** [comment]
I have tried the new metainfo2.txt but sadly it fails at exactly the same point 35-36%

Any other ideas?

**Reply by harman-f** [reply]
Not really.
There are no file checks included into the metainfo2.

Looks like the file or partition is corrupt. Only guessing here.

**Comment by cirrussr20** [comment]
OK, thanks for your help. Is there any way to format the partition? It seems odd that other uploads like maps work fine.

**Reply by harman-f** [reply]
sure, but I would not do it just blindly.

Do you have a D-Link?
Would be better to get into unit and test a few things, to understand what is going on here.

**Comment by cirrussr20** [comment]
I do have one somewhere, will dig it out. I am away for a few days so will respond next week on my return. Tx for your help.

Mike

**Reply by harman-f** [reply]
let's do it like that!
I will also test a bit on my end.

**Comment by cirrussr20** [comment]
Hi again – ignore previous, I now have a login prompt, will let you know when I am back next week!

[A picture containing text, screenshot, computer, software  Description automatically generated]

From: harman-f ***@***.***>
Reply to: "Mr-MIBonk/M.I.B._More-Incredible-Bash" ***@***.***>
Date: Tuesday, 16 May 2023 at 23:15
To: "Mr-MIBonk/M.I.B._More-Incredible-Bash" ***@***.***>
Cc: Mike Jones ***@***.***>, Author ***@***.***>
Subject: Re: [Mr-MIBonk/M.I.B._More-Incredible-Bash] Gracenote Update V17 to V21/V22 (Discussion #211)

WARNING: This email is from an EXTERNAL SENDER. DO NOT CLICK links or attachments unless you recognise the sender and the email address, and know the content is safe.

let's do it like that!
I will also test a bit on my end.

—
Reply to this email directly, view it on GitHub<https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/discussions/211#discussioncomment-5921279>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AQSQ6NF7ELM3YDQM3MLEDFLXGP37DANCNFSM6AAAAAAYDP7R54>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

Mike Jones
Technical account Manager
Arrow Business Communications Ltd
0330 440 4449
07721575799
www.arrowcommunications.co.uk
Read Arrow's blogs here
Registered address: Arrow Business Communications, The Wharf, Abbey Mill Business Park, Lower Eashing, Godalming, GU7 2QN. Registered in England. No.05643503. Arrow Business Communications Limited is part of the Arrow group of companies.
Email Legal Disclaimer and Confidentiality Statement

**Comment by cirrussr20** [comment]
Hi again – back from my trip, can you describe the best approach to login/check?


From: harman-f ***@***.***>
Reply to: "Mr-MIBonk/M.I.B._More-Incredible-Bash" ***@***.***>
Date: Tuesday, 16 May 2023 at 23:15
To: "Mr-MIBonk/M.I.B._More-Incredible-Bash" ***@***.***>
Cc: Mike Jones ***@***.***>, Author ***@***.***>
Subject: Re: [Mr-MIBonk/M.I.B._More-Incredible-Bash] Gracenote Update V17 to V21/V22 (Discussion #211)

WARNING: This email is from an EXTERNAL SENDER. DO NOT CLICK links or attachments unless you recognise the sender and the email address, and know the content is safe.

let's do it like that!
I will also test a bit on my end.

—
Reply to this email directly, view it on GitHub<https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/discussions/211#discussioncomment-5921279>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AQSQ6NF7ELM3YDQM3MLEDFLXGP37DANCNFSM6AAAAAAYDP7R54>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

Mike Jones
Technical account Manager
Arrow Business Communications Ltd
0330 440 4449
07721575799
www.arrowcommunications.co.uk
Read Arrow's blogs here
Registered address: Arrow Business Communications, The Wharf, Abbey Mill Business Park, Lower Eashing, Godalming, GU7 2QN. Registered in England. No.05643503. Arrow Business Communications Limited is part of the Arrow group of companies.
Email Legal Disclaimer and Confidentiality Statement

**Comment by cirrussr20** [comment]
Ready if/when you are?

From: harman-f ***@***.***>
Reply to: "Mr-MIBonk/M.I.B._More-Incredible-Bash" ***@***.***>
Date: Tuesday, 16 May 2023 at 23:15
To: "Mr-MIBonk/M.I.B._More-Incredible-Bash" ***@***.***>
Cc: Mike Jones ***@***.***>, Author ***@***.***>
Subject: Re: [Mr-MIBonk/M.I.B._More-Incredible-Bash] Gracenote Update V17 to V21/V22 (Discussion #211)

WARNING: This email is from an EXTERNAL SENDER. DO NOT CLICK links or attachments unless you recognise the sender and the email address, and know the content is safe.

let's do it like that!
I will also test a bit on my end.

—
Reply to this email directly, view it on GitHub<https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/discussions/211#discussioncomment-5921279>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AQSQ6NF7ELM3YDQM3MLEDFLXGP37DANCNFSM6AAAAAAYDP7R54>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

Mike Jones
Technical account Manager
Arrow Business Communications Ltd
0330 440 4449
07721575799
www.arrowcommunications.co.uk
Read Arrow's blogs here
Registered address: Arrow Business Communications, The Wharf, Abbey Mill Business Park, Lower Eashing, Godalming, GU7 2QN. Registered in England. No.05643503. Arrow Business Communications Limited is part of the Arrow group of companies.
Email Legal Disclaimer and Confidentiality Statement

**Reply by BooBoo124** [reply]
Hi sorry to Hijack. But I've updated gracenote from V22 to V23 now I have no album art at all. Is there a way to down grade? If I try with V22 on the red menu it will not allow it instead of (Y) it shows (D) 
Can the info text be edit so as not to check the version number?

**Reply by cirrussr20** [reply]
What car, what system, all the others fail about 1/3 of the way through - will try V23 and see what happens. I just do it through maintenance menu?

**Reply by Schmisoo** [reply]
> What car, what system, all the others fail about 1/3 of the way through - will try V23 and see what happens. I just do it through maintenance menu?
> 
> 

https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/issues/357

**Comment by BooBoo124** [comment]
Audi A4 MHI2Q_ER_AUG22_P4212 MU1177.

I have seen post from a couple of people that have had no gracenote working after V23 update. The version screen shows its there but it appears to be dead

**Comment by cirrussr20** [comment]
OK I have two cars - R8 MHI2_ER_AU62X_P5099_MU1340 and also A6 MHI2_ER_AUG11_P0040_MU0355 will try both and report back

**Reply by Schmisoo** [reply]
> OK I have two cars - R8 MHI2_ER_AU62X_P5099_MU1340 and also A6 MHI2_ER_AUG11_P0040_MU0355 will try both and report back

https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/issues/357

Same issue

**Comment by BooBoo124** [comment]
Did not install it or  no covers?

**Comment by cirrussr20** [comment]
Worked on A6 MHI2_ER_AUG11_P0040_MU0355 and shows 2023 album covers. R8 not here at moment will report back when it's back

**Comment by BooBoo124** [comment]
That's odd why it killed gracenote on mine

**Comment by cirrussr20** [comment]
Might need Harman-F to help with that - on the R8 all the grace note updates fail at 35%???

**Comment by BooBoo124** [comment]
I don't think @Harman-F has been on for a bit 😕

**Comment by cirrussr20** [comment]
What happens if you try to load V22 back? I don't use the Red Menu, just use system update via maintenance menu?

**Comment by BooBoo124** [comment]
It won't allow a roll ack install via red menu. Gonna try mmi setting install menu later

**Comment by BooBoo124** [comment]
I'd have thought gracenote could have been done via M.I.B but I can't see where if so

**Comment by BooBoo124** [comment]
It's allowing me to install V21 via the system update menu! And that has worked. Might try V23 again via this method

**Comment by cirrussr20** [comment]
Sounds positive - interesting to see what it does

**Comment by BooBoo124** [comment]
V23 still doesn't work 
Trying V22 now


V22 works fine. I think I'll stick at that

**Reply by Schmisoo** [reply]
Can confirm! Updated from v17 to v22 on MHI2_ER_AU37x_P5089_MU1326 (A3). Album Covers are still displayed. Obviously the v23 has some issues.

**Comment by Ash051201** [comment]
Hi all, was wondering if anyone has the gracenote2 aio file for eu? 

Kind regards,
Ash
